### PR TITLE
704 type marker back data

### PR DESCRIPTION
**Pull request by Jday7879**
# Pull Request Title

# Summary

Add your summary here - keep it brief, to the point, and in plain English.

# Type of Change

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe):

# Checklists

This pull request meets the following requirements:

## Creator Checklist

- [x] Installable with all dependencies recorded
- [x] Runs without error
- [x] Follows PEP8 and project-specific conventions
- [x] Appropriate use of comments, for example, no descriptive comments
- [x] Functions documented using Numpy style docstrings
- [x] Assumptions and decisions log considered and updated if appropriate
- [x] Unit tests have been updated to cover essential functionality for a reasonable range of inputs and conditions
- [x] Other forms of testing such as end-to-end and user-interface testing have been considered and updated as required

If you feel some of these conditions do not apply for this pull request, please
add a comment to explain why.

## Reviewer Checklist

- [x] Test suite passes (locally as a minimum)
- [x] Peer reviewed with review recorded

# Additional Information

Please provide any additional information or context that would help the reviewer understand the changes in this pull request.

# Related Issues

Link any related issues or pull requests here.

### CODE REVIEW

**Comment by Jday7879** [comment]
@robertswh this code now passes unit test data. It involved removing and the back data and re applying a couple of times. It might be possible to do this alternate ways, but this currently works for back data.

**Comment by robertswh** [comment]
@Jday7879 very nice. I think that's how I imagined it would work. I think this works as a bug fix and will require a more substantial refactor to simplify the code.

**Review by lhubbardONS — APPROVED** [review]
Code all looks good, happy to approve changes
